tests/test_local_json.py

```diff
@@ -135,6 +135,19 @@ def test_graph():
     assert s1.pred == 'hasRelatedSynonym'
     assert s1.xrefs == ['GOC:mah']
 
+    GOSLIM = 'goslim_generic'
+    subsets = ont.subsets(NUCLEUS)
+    print("SUBSETS: {}".format(subsets))
+    assert GOSLIM in subsets
```

Comment:
Make a separate ticket for this. We should align across all our code. 

FWIW I've been happy with plain pytest; but it can be combined with unittest https://docs.pytest.org/en/latest/unittest.html